service/src/main/java/com/oms/util/Logger.java

```diff
@@ -29,4 +29,4 @@ public void log(String msg) {
             throw new RuntimeException(e);
```

Comment:
I think it would have been better not to continue to throw a RuntimeException, but to print the error message and stack trace to stderr and continue.